lib/screens/signup_page.dart

```diff
@@ -56,107 +61,183 @@ class _SignupPageState extends State<SignupPage> {
   }
 
   Widget _form(BuildContext context) {
-    return Form(
+    return Container(
+      child: Form(
         key: _formKey,
         child: Column(
-          crossAxisAlignment: CrossAxisAlignment.center,
           children: <Widget>[
+            SizedBox(height: 40.0),
             TextFormField(
               keyboardType: TextInputType.emailAddress,
-              decoration: const InputDecoration(
-                hintText: 'Email',
+              decoration: InputDecoration(
+                labelText: 'Email',
+                filled: true,
+                fillColor: Color(0xF6F6F6F6),
+                enabledBorder: OutlineInputBorder(
+                  borderSide: BorderSide(
+                    color: Colors.grey[200],
+                  ),
+                ),
+                prefixIcon: Icon(Icons.email),
               ),
               validator: (value) {
                 if (value.isEmpty) {
-                  return 'Email needed';
+                  return 'Email required.';
                 } else if (!_emailValid.hasMatch(value)) {
-                  return 'You must enter a valid email address';
+                  return 'Please enter a valid email address.';
                 }
```

Comment:
There also must be a case when "Account associated with this email already exists". I believe firebase exception handler file has this messages already.